2024-05-24 07:48:43,782 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-24 07:48:43,782 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Configure stats pid to 18128
2024-05-24 07:48:43,782 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/.config/wandb/settings
2024-05-24 07:48:43,782 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/pytorch/Projects/SAE_vs_DAS/wandb/settings
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/Users/maheepchaudhary/pytorch/Projects/SAE_vs_DAS/main.py', 'program': '/Users/maheepchaudhary/pytorch/Projects/SAE_vs_DAS/main.py'}
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_init.py:_log_setup():520] Logging user logs to /Users/maheepchaudhary/pytorch/Projects/SAE_vs_DAS/wandb/run-20240524_074843-b0qpy5ck/logs/debug.log
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/maheepchaudhary/pytorch/Projects/SAE_vs_DAS/wandb/run-20240524_074843-b0qpy5ck/logs/debug-internal.log
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_init.py:init():560] calling init triggers
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_init.py:init():610] starting backend
2024-05-24 07:48:43,783 INFO    MainThread:18128 [wandb_init.py:init():614] setting up manager
2024-05-24 07:48:43,785 INFO    MainThread:18128 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-05-24 07:48:43,788 INFO    MainThread:18128 [wandb_init.py:init():622] backend started and connected
2024-05-24 07:48:43,792 INFO    MainThread:18128 [wandb_init.py:init():711] updated telemetry
2024-05-24 07:48:43,814 INFO    MainThread:18128 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-24 07:48:43,914 WARNING MainThread:18128 [wandb_init.py:init():1168] interrupted
Traceback (most recent call last):
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1164, in init
    return wi.init()
           ^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 747, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-05-24 07:48:47,367 WARNING MsgRouterThr:18128 [router.py:message_loop():77] message_loop has been closed
